store/search/src/main/scala/org/apache/spark/rpc/Master.scala

```diff
@@ -80,9 +96,18 @@ class Master(sparkConf: SparkConf, port: Int) {
           rpcEnv = new NettyRpcEnvFactory().create(config)
           val registryEndpoint: RpcEndpoint = new Registry(rpcEnv, Master.this)
           rpcEnv.setupEndpoint("registry-service", registryEndpoint)
+          if (!isStarted) {
+            synchronized {
+              if (!isStarted) {
+                flag = true
+              }
+            }
+          }
           rpcEnv.awaitTermination()
         }
       }).start()
+    } else {
+      LOG.info("rpcEnv exists:" + rpcEnv.address)
```

Comment:
What does it mean?
Does it mean the search mode master has already started or something wrong?
Better to optimize the log, for example
```
Search mode master has already been started.
``
OR 
```
Address for search mode master has been used by other applications, start failed.
```

Reply:
Search mode master has already existed before.  maybe by this search mode program, maybe by others.